packages/react-native-web/src/exports/StyleSheet/compile.js

```diff
@@ -144,6 +144,16 @@ function createAtomicRules(identifier: string, property, value): Rules {
       break;
     }
 
+    case 'scrollIndicator': {
+      if (value === 'none') {
+        rules.push(
+          `${selector}::-webkit-scrollbar{width: 0 !important}`,
```

Comment:
Why is this rule `width:0` and using `!important`? Would `display:none` work instead?